cardano-cli/src/Cardano/CLI/Types/Errors/AddressInfoError.hs

```diff
@@ -0,0 +1,14 @@
+module Cardano.CLI.Types.Errors.AddressInfoError
```

Comment:
Does this need a separate module?

Reply:
I put it separate from the command error because I think command errors should be separate from domain errors.  I'm wondering if we should have a separate module for command errors.

Reply:
Merging now.  Can refactor later.